app/client/cypress/integration/national.spec.js

```diff
@@ -0,0 +1,188 @@
+
+    const text = /^Biological condition tells us how healthy a waterbody is/;
+
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+
+    // Click "Expand All" and verify that "Collapse All" is now visible
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('not.exist');
+
+    cy.findByText(text)
+      .filter(':visible')
+      .should('exist');
+
+    // Click "Collapse All" and verify that "Expand All" is now visible and accordion text is hidden
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .click();
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Collapse All')
+      .should('not.exist');
+    cy.findAllByTestId('hmw-expand-collapse')
+      .filter(':visible')
+      .contains('Expand All')
+      .should('exist');
+    cy.findByText(text).should('not.exist');
+  });
+
+  it('Clicking the Nutrient Pollution link opens a new tab', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'href', 'https://www.epa.gov/nutrientpollution');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'target', '_blank');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking the “Learn more about what EPA is doing to reduce nutrient pollution” link opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/nutrient-policy-data/what-epa-doing-reduce-nutrient-pollution',
+    );
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'target', '_blank');
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'rel', 'noopener noreferrer');
```

Comment:
I realized `.should()` returns its subject, so you can chain subsequent `.should()` calls to simplify your code:

```suggestion
    cy.findByText(
      /Learn more about what EPA is doing to reduce nutrient pollution/,
    )
      .should(
        'have.attr',
        'href',
        'https://www.epa.gov/nutrient-policy-data/what-epa-doing-reduce-nutrient-pollution',
      )
      .should('have.attr', 'target', '_blank')
      .should('have.attr', 'rel', 'noopener noreferrer');
```